src/main/java/com/habicus/core/entities/User.java

```diff
@@ -0,0 +1,71 @@
+package com.habicus.core.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * User entity that will reference
+ */
+@Entity
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name="user")
+public class User {
+
+  @Id
+  @GeneratedValue
+  private Long id;
```

Comment:
Should we use something other than Long for ID.. UUID (https://docs.oracle.com/javase/6/docs/api/java/util/UUID.html)

Reply:
I'm willing to consider this, but the cons in this thread seem to make me still lean more towards Long.. https://stackoverflow.com/questions/45399/advantages-and-disadvantages-of-guid-uuid-database-keys

- Easier to sort by ID
- Easier to verbalize
- Shorter URL paths

Trying to figure out what benefits we would have in our case using UUID, any ideas?